guide/C/ch_reports.xml

```diff
@@ -755,29 +755,38 @@ an empty report will be generated.</para>
       basic transaction report format. To create this report:
       </para>
       <orderedlist>
-        <listitem><para>Open the report options, from the
-        <emphasis>Accounts</emphasis> tab, click the Expense
-        placeholder account and <emphasis>Select Children</emphasis>.</para></listitem>
-        <listitem><para>From the <emphasis>General</emphasis> tab,
-        Choose relative dates “Start of Previous Year” and “End of
-        Previous Year”.</para></listitem>
-        <listitem><para>From the <emphasis>Sorting</emphasis> tab,
-        set <emphasis>Primary Key</emphasis> to Accounts, and enable
-        <emphasis>Primary Key Subtotal</emphasis>.
-        </para></listitem>
-        <listitem><para>Set the <emphasis>Secondary Key</emphasis> to
-        Date, and set the <emphasis>Secondary Key Subtotal</emphasis>
-        to None. </para></listitem>
+        <listitem>
+          <para>Open the report options, from the
+          <emphasis>Accounts</emphasis> tab, click the Expense
```

Comment:
`<guilabel>Accounts</guilabel>`

pls. check other occurrences of `<emphasis>`.
A incomplete list of gui elements:
accel, guibutton, guiicon, guimenu, guimenuitem, guisubmenu, keycap, keycode, keycombo, keysym, menuchoice, mousebutton, shortcut.

See e.g. https://tdg.docbook.org/tdg/5.0/docbook.html